packages/twenty-front/src/modules/object-record/advanced-filter/components/AdvancedFilterViewFilterValueInput.tsx

```diff
@@ -0,0 +1,65 @@
+import { ConfigurableFilterDropdownContent } from '@/object-record/object-filter-dropdown/components/ConfigurableFilterDropdownContent';
+import { useFilterDropdown } from '@/object-record/object-filter-dropdown/hooks/useFilterDropdown';
+import { FilterDefinition } from '@/object-record/object-filter-dropdown/types/FilterDefinition';
+import { SelectControl } from '@/ui/input/components/SelectControl';
+import { Dropdown } from '@/ui/layout/dropdown/components/Dropdown';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { ADVANCED_FILTER_DROPDOWN_ID } from '@/views/constants/AdvancedFilterDropdownId';
+import { ViewFilter } from '@/views/types/ViewFilter';
+
+interface AdvancedFilterViewFilterValueInputProps {
+  viewFilter: ViewFilter;
+  filterDefinition: FilterDefinition;
+  isDisabled?: boolean;
+}
+
+export const AdvancedFilterViewFilterValueInput = (
+  props: AdvancedFilterViewFilterValueInputProps,
```

Comment:
Please apply the same pattern as the codebase : props spreading and type not interface for props